sphinxcontrib/confluencebuilder/translator.py

```diff
@@ -1826,6 +1846,16 @@ def _start_ac_link_body(self, node):
         Returns:
             the content
         """
+
+        if self.add_secnumbers and node.get('secnumber'):
+            return  self._start_tag(
+                        node,
+                        'ac:link-body',
+                        suffix=(
```

Comment:
@jdknight Using `suffix` here gives the desired result, even though the section number is technically a prefix. Is this expected?

Reply:
The `suffix` argument is for adding trailing content for a tag being created by a `_start_tag` call (not the content which may be wrapped by tags). Typically, this is for only for specifying newlines for formatting niceties in the output (with the exception of character encoding (`CDATA`) use cases). Real content, like the section numbering, should really be added outside the tag-creation call since the section numbering has nothing to do with the tag itself.

While this change does address adding appropriate prefix content inside generated toctree's, adding this logic into the `_start_ac_link_body` helper call would not be the right place for this. A helper method like `_start_ac_link_body` is much like `_start_tag` which focuses on the "proper" creation of the tag portion for a specific use case, and should not really focus on the content of the data.

I believe a better location for this logic would be inside `_visit_reference_intern_uri`. For example:

```
    def _visit_reference_intern_uri(self, node):
        ...
        self.body.append(self._start_ac_link_body(node))
        if self.add_secnumbers and node.get('secnumber'):
            self.body.append('.'.join(map(str, node['secnumber'])) +
                             self.secnumber_suffix)
        self._reference_context.append(self._end_ac_link_body(node))
        ...
```